bgpd/bgp_nb_config.c

```diff
@@ -5198,7 +5198,8 @@ int bgp_neighbors_unnumbered_neighbor_neighbor_remote_as_remote_as_type_destroy(
 		peer = peer_lookup_by_conf_if(bgp, peer_str);
 
 		/* remote-as set to 0 and as_type to unspecified */
-		peer_as_change(peer, 0, AS_UNSPECIFIED);
+		if (peer)
```

Comment:
do you have a crash that you are seeing?  What lead you to this?

Reply:
and you have a coverity license.  Got it.

Reply:
yeah, coverity :)